packages/web-runtime/src/composables/capability/useCapability.ts

```diff
@@ -0,0 +1,34 @@
+import { Store } from 'vuex'
+import get from 'lodash-es/get'
+import { computed, ComputedRef } from '@vue/composition-api'
+import { useStore } from 'web-pkg/src/composables'
+
+export const useCapability = <T>(
+  store: Store<any>,
+  name: string,
+  defaultValue?: T
+): ComputedRef<T> => {
+  return computed((): T => {
+    if (defaultValue) {
```

Comment:
not necessary if you apply the other suggestion in this file

Reply:
Ah, yes, you are totally right.